pkgs/data/fonts/powerline-fonts/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "powerline-fonts-39c99c";
```

Comment:
sorry, just noticed this. you should add a relevant date version, like 2015-04-24. the rationale I believe is that seeing the package name should give you an idea on how old the package is, a short hash doesn't allow that